astrogrism/HST/reference_file_generators/generate_wavelengthrange.py

```diff
@@ -13,12 +15,11 @@
 from .common import common_reference_file_keywords
 
 
-def create_tsgrism_wavelengthrange(outname="wfc3_tsgrism_wavelengthrange.asdf",
-                                   history="WFC3 TSGrism wavelengthrange",
-                                   author="STScI",
-                                   wavelengthrange=None,
-                                   extract_orders=None):
-    """Create a wavelengthrange reference file for WFC3 TSGRISM mode.
+def create_grism_wavelengthrange(grism, chip=None,
+                                 outname="wfc3_grism_wavelengthrange.asdf",
```

Comment:
In the similar direciton as your other changes, I think it's appropriate now that we have the other instruments to change the `wfc3` default to just `hst` default? Or maybe even just drop it?

``` suggestion
                                 outname="hst_grism_wavelengthrange.asdf",
```

Reply:
Good call. Rather than having a default it's probably better to just generate the right name based on the grism selected... I can code that in, give me a minute...